modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/_dnd_table.scss

```diff
@@ -0,0 +1,180 @@
+.dnd-thead {
+	display: table-header-group;
+
+	.fixed & {
+		display: block;
+	}
+}
+
+.dnd-tbody {
+	display: table-row-group;
+
+	.fixed & {
+		display: block;
+	}
+}
+
+.dnd-td,
+.dnd-th {
+	border-right: 1px solid $border-color;
+	display: table-cell;
+	opacity: 0;
+	overflow: hidden;
+	padding: $spacer * 0.75;
+	transition: opacity 0.1s ease-in;
+	white-space: normal;
+
+	.fixed & {
+		opacity: 1;
+	}
+
+	&:last-child {
+		border-right: none;
+	}
+
+	&.item-actions,
+	&.item-selector {
+		width: 1%;
+	}
+
+	&.expand {
+		min-width: 150px;
+		width: 20%;
+	}
+
+	.fixed & {
+		align-items: center;
+		display: flex;
+		min-width: auto;
+	}
+}
+
+.dnd-th {
+	font-weight: 600;
+	position: relative;
+}
+
+.dnd-tr {
+	border-bottom: 1px solid $border-color;
+	display: table-row;
+	min-width: 100%;
+	position: relative;
+
+	.dnd-tbody &:last-child {
+		border-bottom: none;
+	}
+
+	&.with-bg {
+		background-color: $gray-100;
+	}
+
+	&::before {
+		background: var(--primary);
+		bottom: 0;
+		content: '';
+		left: 0;
+		position: absolute;
+		top: 0;
+		transform: scaleX(0);
+		transition: transform 0.1s ease-in;
+		width: 3px;
+	}
+
+	&.active::before {
+		transform: scaleX(1);
+	}
+
+	.fixed & {
+		display: flex;
+		width: fit-content;
+	}
+
+	&.nested + &.dnd-tr:not(.nested) {
+		border-top: 1px solid $border-color;
+	}
+
+	&.nested {
+		border-bottom: 1px solid var(--white);
+		border-width: 0;
+		margin: 0;
+		overflow: hidden;
+		position: relative;
+
+		&.last {
+			border-bottom: none;
+			border-radius: 0 0 10px 10px;
+			margin-bottom: 8px;
+
+			.dnd-td:first-child {
+				border-bottom-left-radius: 10px;
+			}
+
+			.dnd-td:last-child {
+				border-bottom-right-radius: 10px;
+			}
+		}
+
+		.dnd-td {
+			background-color: #f0f5ff;
```

Comment:
Can use `$table-list-active-bg` for this color. I'm assuming if we import atlas-variables.